01-make_graph.py

```diff
@@ -101,6 +101,7 @@ def parsed_meta_yaml(text):
         'url': yaml_dict['source']['url'],
         'req': req,
         'time': time.time(),
+        'meta_yaml': text,
```

Comment:
Can we also store the loaded `meta.yaml`? Otherwise we need to parse the text every time we want information (eg what are all the recipes I maintain).

Reply:
both or just the loaded one? I need the raw text for the other PR.

Reply:
Both is good with me. (I'd rather store the information than to keep querying for it)